pkg/summarizer/summary.go

```diff
@@ -750,39 +749,37 @@ func firstFilled(values ...int32) int {
 }
 
 const (
+	// TODO(chases2): deprecate and remove (migrated to tabulator)
 	includeFilter = "include-filter-by-regex"
 	excludeFilter = "exclude-filter-by-regex"
-	// TODO(fejta): others, which are not used by testgrid.k8s.io
 )
 
-// filterGrid truncates the grid to rows with recent results and matching the white/blacklist.
-func filterGrid(baseOptions string, rows []*statepb.Row, recent int) ([]*statepb.Row, error) {
-
-	vals, err := url.ParseQuery(baseOptions)
-	if err != nil {
-		return nil, fmt.Errorf("parse %q: %v", baseOptions, err)
-	}
+// filterGrid truncates the grid to rows with recent results and, optionally, matching an allow/exclude regex.
+func filterGrid(baseOptions string, rows []*statepb.Row, recent int, useRegexFilter bool) ([]*statepb.Row, error) {
 
 	rows = recentRows(rows, recent)
 
 	rows = filterMethods(rows)
 
-	for _, include := range vals[includeFilter] {
-		if rows, err = includeRows(rows, include); err != nil {
-			return nil, fmt.Errorf("bad %s=%s: %v", includeFilter, include, err)
+	if useRegexFilter {
```

Comment:
nit: Maybe `if !useRegexFilter { return rows, nil }` to limit what's inside the conditional?

Reply:
Good idea; done